module/actors/sheets/BWCharacterSheet.ts

```diff
@@ -84,7 +84,7 @@ export class BWCharacterSheet extends BWActorSheet<CharacterSheetData, BWCharact
                     const s = i as SkillDataRoot;
                     if (s.data.learning) {
                         learning.push(s);
-                    } else if (s.data.training && ["martial", "training"].includes(s.data.skilltype)) {
+                    } else if (s.data.training) {
```

Comment:
This should not be in this PR, as it is a change related to a different issue.

If submitting multiple PRs, please try to have each one based off of the current master, not each other.